papyri/myst_ast.py

```diff
@@ -99,6 +99,26 @@ class MParagraph(Node):
     # data: Any
 
 
+@register(4052)
+class MList(Node):
+    type = "list"
+    children: List["ListContent"]
+    ordered: bool = False
+    # start: int
+    # spread: bool
```

Comment:
One questions is are spread and start necessary for myst to render ?

Reply:
No, I don't think so, from what I have seen so far.